subgenerator/index.js

```diff
@@ -1,25 +1,25 @@
 'use strict';
 var path = require('path');
-var generators = require('yeoman-generator');
+var Generator = require('yeoman-generator');
 var superb = require('superb');
 
-module.exports = generators.Base.extend({
-  constructor: function () {
-    generators.Base.apply(this, arguments);
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
 
-    this.argument('namespace', {
+    this.option('name', {
```

Comment:
Actually that's pretty simple. It's breaking because generator-node is pre 1.0, so it doesn't recognize the arguments being passed as part of the options.

We should fix that in `yeoman-generator` so we keep compatibility with older generators too.